plugins/review-agent-governance/test/run-tests.sh

```
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+# run-tests.sh — guard review-agent-governance.cedar against the in-on-String
+# forbid bug (#598). Part A (grep) always runs; Part B (cedar validate) runs
+# only if the `cedar` CLI is installed, else SKIPs without failing.
+# Exit codes: 0 pass, 1 fail.
+set -uo pipefail
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+cd "$SCRIPT_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)